### ISSUE TITLE
Replace heavy stick in spear recipes with long stick

### ISSUE THREAD

**Issue by kevingranade** [CleverRaven/Cataclysm-DDA#24017]
Recent discussions around spears led me to reexamine spear crafting.
I noticed that all spears are made from the "heavy stick" item, which is dropped from a myriad of sources, very few of which would produce a 6' straight pole.

Proposal is:
- [ ] Add a "long stick" item
- [ ] have it drop in suitable locations. e.g. smashing a young tree would drop one, cutting down an adult tree would produce a number of them, etc.
- [ ] Replace "heavy stick" in spear recipes with "long stick".
- [ ] Make long stick disassemble into two heavy sticks.

Possibly we would instead have an intermediate item "pole" consumed by the spear recipes that is produced from the long stick via carving.
Suggestions for other sources of long sticks or poles are welcome.

**Comment by 0be1isk** [comment]
Fundamentally, a spear is really just a long stick with a metal tip. I welcome this "long stick" idea. We could possible introduce an item such as A "spear tip" that is crafted from metal scrap, and requires welding and hammering to forge. And once you have a "long stick" and a "spear tip" then you can craft a "sturdy spear"...this is just spit balling ideas, but I think there is a nice compromise with this idea. You're gonna need an items with hammering (easy to get)  and welding (not easy to get) to make spears. This prevents early game kiting since you don't have a spear, and need to scavenge for the supplies to make one in the first place. But it also eliminates the need for the pointless "fragile" tag as well

**Comment by mlangsdorf** [comment]
I think there's space for not very durable spears, made from fire-hardened wood points or spikes hammered into a long stick and then resharpened.

Makeshift weapons that are barely adequate should be in the game. Duct taping a knife blade to a broom handle is the kind of thing that someone might do if they needed a weapon and couldn't find anything better.

**Comment by DemAvalon** [comment]
I'm used to using slender trunks in unrealworld, so I think it's a good idea
 
I heavily disagree with 0be1isk's idea, it would become necessary to do this to every weapon, you would have to craft the sword blade then the sword handle to make an arming sword, you would have to craft the axe handle, then the axe head for the axes, and of course different swords have different blades and handles, so the crafting menu would be filled with an enormous amount of X blades and X handles, just keep as is (in the way things are crafted) and just make it so the time it takes to craft takes into consideration shaping the raw ingredients into the respective parts and assembling them.

**Comment by 0be1isk** [comment]
@DemAvalon I wouldn't say it would be necessary for all items. Obviously you would be able to find complete axe's around the world, as well as swords. This would simply apply to makeshift items (which makes complete sense). The idea wasn't to completely overhaul the crafting system, but rather make it more of a challenge to create makeshift spears. It's a decent compromise between the people who think spears are too OP for beginners, and those (like myself) who think spears are fine the way they are and do what they were meant to do perfectly. So if i happen to find anything that gives me welding, I'm in luck. I'd say it's having your cake, and eating it too.

**Comment by kevingranade** [comment]
> A "spear tip" that is crafted from metal scrap, and requires welding and hammering to forge.

Once you're messing with forging, that's a whole different thing. Can you cold-hammer iron scrap into a serviceable spearhead? Seems likely, but also seems like it would take a long time and require some skill, especially to fasten it to the spear.

> But it also eliminates the need for the pointless "fragile" tag as well.

That doesn't follow, unless you're also eliminating the other primitive spears?

> Duct taping a knife blade to a broom handle is the kind of thing that someone might do if they needed a weapon and couldn't find anything better.

Yes, but I don't see how that has anything to do with the current discussion since it wouldn't have reach...

As for subassemblies vs recipes that list all their tools and components, the latter has pretty solidly won out.

> make it more of a challenge to create makeshift spears. 

This doesn't have anything to do with whether you craft a "spear head" item and then use it to craft a "spear" item, the only thing at issue is what tools, components and skills are required.

**Comment by pipehat** [comment]
I think this is a nightmare of oversimulation and just adds useless complexity. Please don't.

**Comment by 0be1isk** [comment]
I mean, imagine if we simulated needing to take a piss or taking a dump. Now THAT'S oversimulating. This, not so much

**Comment by pipehat** [comment]
Actually I suspect you'd find more popular support for waste management simulation than this -- although I'm not interested in that either.

**Comment by RadHazard** [comment]
>>Duct taping a knife blade to a broom handle is the kind of thing that someone might do if they needed a weapon and couldn't find anything better.
>
>Yes, but I don't see how that has anything to do with the current discussion since it wouldn't have reach...

How long are spears supposed to be in this game?  I was always under the impression that they were roughly the same height as a person (with a few exceptions like the awl pike), which would make a broom or mop handle just about the right length for a makeshift spear.

If you're going to add long sticks to the game, I'd think brooms and mops should also be usable as a handle substitute (at least for the early makeshift variants).  They're not nearly as common as regular sticks, but they would offer a handle substitute for anyone stuck in a city in the early game.

**Comment by kevingranade** [comment]
> How long are spears supposed to be in this game? I was always under the impression that they were roughly the same height as a person

I was thinking about a "sweeping" broom, which is about chest high with the brush, so probably 4' or less for the handle, but it looks like pushbrooms have 5' or more long handles, so that would qualify.

**Comment by NickGrease** [comment]
> very few of which would produce a 6' straight pole

It could be a neat little mechanic, with a hammer and an ax, you could try to split a two by four in half, producing 0,1, 2 rough poles or just splinters, then clean up the edges with a knife.

**Comment by BorkBorkGoesTheCode** [comment]
If long sticks are added to the game, could they replace the heavy stick in curtain windows? Using MShock and seeing windows drop leafy branches is a weird experience.

**Comment by Blank99** [comment]
If the heavy stick becomes too short, and disasembling a long one produces two, wouldn't this pretty much result in the obvious makeshift recipe of "stick+stick+tip+duct tape/thread"?
Or still using one heavy stick, but producing non-reach spears, short spears, if you will?

You would then have the choice between "fast stabby weapon", "crappy reach weapon", and "better reach weapon that requires rarer resources".

I would be sad seeing being able to affix a pointy end to a stick become scarcer on the early game.

**Comment by kevingranade** [comment]
> wouldn't this pretty much result in the obvious makeshift recipe of "stick+stick+tip+duct tape/thread"?

No, because that will break under it's own weight, much less use in combat.
> I would be sad seeing being able to affix a pointy end to a stick become scarcer on the early game.

There is in fact a point stick, the only thing this makes rarer is a pointy stick with reach.

**Comment by Lasiace** [comment]
Just my two cents, but I think there is a danger of this change turning into Arrows 2.0 where you had to craft all the components just for the sake of one thing. So if this change is specifically for just crafting spears, I don't think that this would be a good idea.

On the other hand, if this becomes a basic component used in more recipes, perhaps replacing other recipes where you typically used a heavy stick or two-by-four, then I think there would be a better argument for it. The previous suggestion to replace the heavy sticks currently being produced from man-made sources like windows seems like it would be good as well.

As a minor thing, if this does go through I think it's worth renaming heavy stick to "tree branch" or something similar instead, as long stick and heavy stick would seem too similar, and heavy sticks would be limited to natural sources anyway.

**Comment by glamcube17** [comment]
Making them obtainable from windows and the like would kind of defeat the purpose of making them harder to get than heavy sticks, wouldn't it? You could still just smash a window, smash a locker, knife spear.

The point about seeing leafy branches come from windows seems like a problem with the tileset, not the game itself.

**Comment by Wad67** [comment]
No thanks, don't like the idea of yet another item & intermediate step just for crafting purposes.

Here's my counter proposal:
Keep the current heavy stick we have, change the description if you must.

**Comment by BorkBorkGoesTheCode** [comment]
Could use the cudgel as a long stick as someone suggested.

**Comment by stolencatkarma** [comment]
there's really two types of spears in the 'world' that exist for realism sake.

make_shift and manufactured. those you could make yourself and those you can buy in the store.

so my proposal would be ONLY have a recipe for make_shift spear and manufactured would HAVE to be found. you couldn't make them.

**Comment by JayThirtySeven** [comment]
Not sure exactly where this fits into the discussion but it seems relevant: https://www.amazon.com/dp/B0017234ZS/ref=cm_sw_r_other_apa_7otkBbK7YF6YT

Basically there are a few models of knife where the handle is attached with a standard broom handle thread for the express purpose of making a spear. They also mention using a branch for a handle by using a nail or screw to secure the blade. Not saying these shoupd be in the game, but it gives an idea of how a character might assemble a spear.

Another thought: Cutting and/or sharpening the head of a shovel seems like a quick way to get a spear. Not all shovels would be long enough but some certainly are.

**Comment by Wayticus** [comment]
Sounds good to me,  I'd say also change the recipes for the shovel, hoe, mop, broom, etc to require long sticks, and disassemble into them.

**Comment by glamcube17** [comment]
Aren't the brooms we've got fully plastic? They don't have wood as a material, at least.
